addons/fieldmanual/config.cpp

```diff
@@ -0,0 +1,20 @@
+#include "script_component.hpp"
+
+class CfgPatches
+{
+    class ADDON
+    {
+        name = COMPONENT_NAME;
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {"ace_common"};
+        author = ECSTRING(common,ACETeam);
+        authors[] = {"Hypoxic"};
+        url = ECSTRING(main,URL);
+        VERSION_CONFIG;
+    };
+};
+
+#include "CfgHintCategories.hpp"
+#include "CfgHints.hpp"
```

Comment:
```suggestion
#include "CfgHints.hpp"

```

New lines at end of file.